src/funcCoreTools/validateFuncCoreToolsInstalled.ts

```diff
@@ -44,9 +48,28 @@ export async function validateFuncCoreToolsInstalled(context: IActionContext, me
                 items.push(DialogResponses.learnMore);
             }
 
+            if (os.platform() === 'linux') {
+                const linuxDistroTag: LinuxDistroTag | undefined = await getLinuxDistroTag();
+                if (linuxDistroTag) {
+                    let linuxMessage: string = '';
+                    if (linuxDistroTag['PRETTY_NAME']) {
```

Comment:
Does the pretty name include a version as well? Could you give examples of what the expected output is?

Reply:
Yeah, my observation has been that if there is both a version and pretty_name, the pretty_name will include the version.

See example below: https://github.com/microsoft/vscode-azurefunctions/pull/3459#discussion_r1037639930

Reply:
Also here is the default message when no package manager is detected on Linux

![image](https://user-images.githubusercontent.com/40250218/205188284-53fcc5bd-a2d4-4a99-a762-a1dca14f5ec7.png)